pkg/monitors/collectd/monitorcore.go

```diff
@@ -80,7 +80,7 @@ func (bm *MonitorCore) SetConfigurationAndRun(conf config.MonitorCustomConfig) e
 // SetConfiguration adds various fields from the config to the template context
 // but does not render the config.
 func (bm *MonitorCore) SetConfiguration(conf config.MonitorCustomConfig) error {
-	return bm.collectdInstance().ConfigureFromMonitor(bm.monitorID, bm.Output, bm.UsesGenericJMX)
+	return bm.collectdInstance().ConfigureFromMonitor(bm.monitorID, bm.Output, bm.UsesGenericJMX, conf.MonitorConfigCore().IntervalSeconds)
```

Comment:
This will cause probably the last monitor that uses generic jmx to have its interval configured.  That is a confusing behavior and it would be better to figure out a way to set  an interval for each monitor separately.

Reply:
I don't know how we can resolve this short of having multiple collectd processes. So you will need a collectd process for every java plugins configured with a different interval.